embedded-graphics/src/style/primitive_style.rs

```diff
@@ -199,12 +223,47 @@ where
         self
     }
 
+    /// Sets the stroke alignment.
+    pub fn stroke_alignment(mut self, stroke_alignment: StrokeAlignment) -> Self {
+        self.style.stroke_alignment = stroke_alignment;
+
+        self
+    }
+
     /// Builds the primitive style.
     pub fn build(self) -> PrimitiveStyle<C> {
         self.style
     }
 }
 
+impl<C> From<&PrimitiveStyle<C>> for PrimitiveStyleBuilder<C>
+where
+    C: PixelColor,
+{
+    fn from(style: &PrimitiveStyle<C>) -> Self {
+        Self {
+            style: style.clone(),
+        }
+    }
+}
```

Comment:
Nice addition, it makes the simulator example code a lot cleaner :ok_hand:

Reply:
Thanks for reminding me that I added this, I forgot to add this to the changelog.